app/assets/stylesheets/application.scss

```diff
@@ -13,4 +13,21 @@
  *= require_tree .
  *= require_self
  */
- @import "bootstrap";
+//Importing bootstrap
+@import "bootstrap";
+/* Base */
+:root {
```

Comment:
Consider using CSS variables for defining colors and font sizes. This will make it easier to update the styles in the future and maintain consistency throughout the application.